packages/orbit-components/src/Timeline/TimelineStep/components/TypeIcon.jsx

```diff
@@ -0,0 +1,51 @@
+// @flow
+import * as React from "react";
+
+import CheckCircle from "../../../icons/CheckCircle";
+import Circle from "../../../icons/CircleEmpty";
+import AlertCircle from "../../../icons/AlertCircle";
+import CloseCircle from "../../../icons/CloseCircle";
+import StyledIconWrapper from "../primitives/StyledIconWrapper";
+import type { Type } from "..";
+
+const getTypeIcon = ({ type, active }) => {
+  if (type === "success" && active)
```

Comment:
If it is success and last is it also success and active? If so, it will be animated, but it shouldn't 🤔 what determines if it is active? I'm looking at the _inside modal_ story and the last step has this animation but I feel it should only have the green check, as the process is apparently finished. Is it controllable and it's just a problem on the story? Or is there some logic missing here?

Reply:
yes, it should be controlled, you are right ;) thanks